### DISCUSSION TITLE
how to open command console

### DISCUSSION THREAD

**Discussion by PAIDMAN** [sultim-t/xash-rt#130] [Q&A]
i got this bug 
https://www.reddit.com/r/HalfLife/comments/ianxeq/apprehension_ending_cut_scene_glitch/

and I need the console to resolve it . I try to do it in the original games but the save in the ray traced version cant be load .

**Comment by GrandTheftWalrus** [comment]
I too am wondering how to open the console

**Reply by GrandTheftWalrus** [reply]
Nevermind I just realized that besides xash3d.exe, there's an xash3d-console.bat that can be used to start the game which lets there be a "console" option in the game menu > controls > console
